tools/README.md

```diff
@@ -19,3 +19,45 @@ https://web.archive.org/web/20131210001526/http://xecdesign.com/qemu-emulating-r
 https://github.com/johnlane/rpi-utils/blob/master/kernel/build-kernel-qemu
 
 https://github.com/polaco1782/raspberry-qemu/blob/master/build-kernel-qemu
+
+# Vagrant
```

Comment:
Should not use a single # at this level since it's rendered as h1 or title. At least use ##